src/www/app/app.ts

```diff
@@ -15,8 +15,8 @@
 import * as errors from "../model/errors";
 import * as events from "../model/events";
 import {Server} from "../model/server";
-import {ServerConnectionState} from "../ui_components/server_card";
-import {ServerListItem} from "../ui_components/server_list";
+import {ServerConnectionState} from "../views/servers_view/server_connection_indicator";
```

Comment:
I could re-export these from `servers_view` if that's amenable

Reply:
so it'd be

```
import {ServerListItem, ServerConnectionState} from "../views/servers_view";
```

Reply:
That would be fine with me. We are leaking implementation details and the app shouldn't need to know how the servers list view is implemented.

Reply:
Right, basically my thinking here